client/components/select-dropdown/index.jsx

```diff
@@ -24,13 +24,39 @@ import Count from 'components/count';
  * Module variables
  */
 const { Component, PropTypes } = React;
-var noop = () => {};
+const noop = () => {};
```

Comment:
I prefer to either do `import { noop } from 'lodash'` or use the anon function directly.